nsinit/config.go

```diff
@@ -70,6 +71,22 @@ func modify(config *configs.Config, context *cli.Context) {
 	config.AppArmorProfile = context.String("apparmor-profile")
 	config.ProcessLabel = context.String("process-label")
 	config.MountLabel = context.String("mount-label")
+
+	userns_uid := context.Int("userns-root-uid")
+	maxUid := 1 << 32
```

Comment:
Yeah, want to set it to 2^32. Seems to be working but if it is undefined then need to change this. Thanks.